recipe/meta.yaml

```diff
@@ -20,8 +20,10 @@ requirements:
   run:
     - python >=3.6
     - geopandas >=0.9
-    - dask >=2.2.0
-    - distributed >=2.2.0
+    - dask >=2.2.0,!=2021.05.1
+    - distributed >=2.2.0,!=2021.05.1
+    - numba
+    - pygeos
```

Comment:
Those are actually not yet required for alpha 4, but only used for features add afterwards. Now, since we have quite some changes since then, I can probably do a quick alpha 5

Reply:
Ah I see your comment above about it :) 
Yeah, indeed can't hurt. But I also just created an alpha 5 (https://github.com/geopandas/dask-geopandas/releases/tag/v0.1.0a5), in case you directly want to update it to that